core/src/main/java/org/elasticsearch/index/query/TermsQueryBuilder.java

```diff
@@ -142,72 +283,150 @@ public TermsQueryBuilder disableCoord(boolean disableCoord) {
         return this;
     }
 
+    public boolean disableCoord() {
+        return this.disableCoord;
+    }
+
+    public boolean isTermsLookupQuery() {
```

Comment:
We used to do something a bit more convoluted here. I say we keep it and make it private.